sphinxcontrib/plantuml.py

```diff
@@ -228,12 +229,14 @@ def generate_plantuml_args(self, node, fileformat):
     return args
 
 
-def render_plantuml(self, node, fileformat):
+def render_plantuml(self, node, fileformat, plantuml_type=None):
+    if not plantuml_type:
+        plantuml_type = fileformat
```

Comment:
Instead of adding separate `fileformat, plantuml_type` parameters, maybe `eps:text` can be mapped to `<filename>.text.eps` (i.e. `'text.eps': ['-teps:text']`)?

Suppose `-teps` and `teps:text` generate different contents, the output filenames must be unique.

Reply:
Oh, but it might break batch rendering where the output file names are controlled by plantuml.

If `-teps:text` doesn't break existing users, maybe it's okay to always use `-teps:text`.

Reply:
Should this be an option available on config?
`plantuml_latex_output_format = "eps:text"?

Reply:
> If `-teps:text` doesn't break existing users, maybe it's okay to always use `-teps:text`.

Depends on the definition of "breaking". For sure it can lead to a different output

Reply:
Yeah, that seems good if we need to support both eps formats.

If `eps:text` is strictly better in latex rendering, I think it's good to use it by default. `eps` format wouldn't be used for the other purposes.

Reply:
I have no preference over adding option vs changing the default. If we add an option, it will be something like `plantuml_eps_embed_fonts` so that all `<hash>.eps` files of the same name are rendered with the same configuration.